src/AddIns/Uno.UI.Lottie/WasmScripts/uno-lottie.d.ts

```diff
@@ -14,9 +15,10 @@ declare namespace Uno.UI {
     class Lottie {
         private static _player;
         private static _runningAnimations;
-        static setAnimationProperties(newProperties: LottieAnimationProperties): string;
+        private static _numberOfFrames;
+        static setAnimationProperties(newProperties: LottieAnimationProperties, animationData?: AnimationData): string;
         static stop(elementId: number): string;
-        static play(elementId: number, looped: boolean): string;
+        static play(elementId: number, fromProgress: number, toProgress: number, looped: boolean): string;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class method 'play' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)